snap/info.go

```diff
@@ -117,6 +117,36 @@ func NoneSecurityTag(snapName, uniqueName string) string {
 	return ScopedSecurityTag(snapName, "none", uniqueName)
 }
 
+// DataDir returns the data directory of the snap.
+func DataDir(snapName string, revision Revision) string {
```

Comment:
We use "snapName" here but The caller in s.Info() is using s.InstanceName() - should we try to use the same terminology everyhwere? Was this discussed?

Reply:
I agree it's a bit confusing. The code used to have `snapName` everywhere, then we started using `instanceName` wherever changes were introduced (and was applicable), but in this place either can be passed. Maybe I should just go with `name` instead?

Reply:
Renamed to `name`